Utils/Dataflow/030_PDFAnalyzer/manager.py

```diff
@@ -1769,9 +1772,10 @@ def show_paper_visual(self, window, paper, e=False):
                     else:
                         header_row = False
                         color = "black"
-                    for l in row:
-                        cnvs.create_rectangle((l.left, l.top + 10, l.right,
-                                               l.bottom + 10), outline=color)
+                    for line in row:
+                        cnvs.create_rectangle((line.left, line.top + 10,
+                                               line.right, line.bottom + 10),
+                                              outline=color)
```

Comment:
Align this one properly.

Reply:
@konvpalto,
however strange it might look, it *does* be aligned properly.
The `outline=color` is aligned to the first symbol after `create_rectangle()` opening bracket; previous line is aligned to the one of the first argument tuple. 

Moving the last line to the right would lead to the `pycodestyle` alarm:
```
E127 continuation line over-indented for visual indent
    Continuation lines indentation.

    Continuation lines should align wrapped elements either vertically
    using Python's implicit line joining inside parentheses, brackets
    and braces, or using a hanging indent.

    When using a hanging indent these considerations should be applied:
    - there should be no arguments on the first line, and
    - further indentation should be used to clearly distinguish itself as a
      continuation line.
```

Reply:
@mgolosova's explanation leaves me no room for comments. I can only add a side thought that sometimes pycodestyle's indentation logic looks weird, but not here.

Reply:
Yep, I was wrong here, thanks for revealing my mistake!